NEMO/templates/tool_control/tool_status.html

```diff
@@ -245,6 +245,23 @@ <h4>What would you like to do?</h4>
 				{% endif %}
 			{% endif %}
 		</form>
+		<div style="background-color: #f5f5f5 ; ">
+
+			<div class="row" style="margin:0">
+				{% if tool.description %}
+				<div class="col-md-8">
+					<p> {{tool.description| safe }} </p>
+				</div>
+				{% endif %}
+				{% if tool.image %}
+				<div class="col-md-4">
+					<center><p><img src="{{ tool.image.url }}" class="img-fluid img-thumbnail float-right" style="max-width:300px ;max-height:300px" ></p></center>
+				</div>
+				{% endif %}
+			</div>
+		</div>
```

Comment:
```
{% if tool.description or tool.image %}
	<div style="background-color: #f5f5f5; margin-bottom: 15px">
		<div class="row" style="margin:0; padding: 15px 0">
			{% if tool.description %}
				<div class="col-md-8">
					<p>{{tool.description| safe }}</p>
				</div>
			{% endif %}
			{% if tool.image %}
				<div class="col-md-4">
					<img src="{{ tool.image.url }}" class="img-fluid img-thumbnail" style="max-width:300px ;max-height:300px" >
				</div>
			{% endif %}
		</div>
	</div>
{% endif %}
```

also I would move this to the top of the file, right below `<div class="tab-pane active" id="summary">`

Reply:
I think we should keep the original layout and let the description and image below. If the description is long, the 
"What would you like to do?" menu will not be directly available. What do you think about that ?

Reply:
ok, that makes sense. then change the `<div style="background-color: #f5f5f5; margin-bottom: 15px">` to `<div style="background-color: #f5f5f5; margin-top: 15px">` so it's not glued to the button to enable tool